litr/src/main/java/com/linkedin/android/litr/transcoder/VideoTrackTranscoder.java

```diff
@@ -267,4 +297,10 @@ private int writeEncodedOutputFrame() throws TrackTranscoderException {
 
         return encodeFrameResult;
     }
+
+    @VisibleForTesting
+    @Nullable
+    public FrameDropper getFrameDropper() {
```

Comment:
nip: can you rename this to `createFrameDropper`?

Reply:
I think that you meant renaming `initFrameDropper` -> `createFrameDropper`? This here is just a getter visibly only to tests, it'll return the current `frameDropper` (nullable) object.

I've updated the name of `initFrameDropper` to `createFrameDropper`: [VideoTrackTranscoder.java#L87](https://github.com/linkedin/LiTr/blob/a6d524823017dc2cf30ed85026e7d1ad3af4ace9/litr/src/main/java/com/linkedin/android/litr/transcoder/VideoTrackTranscoder.java#L87)

Reply:
I think that you meant renaming `initFrameDropper` -> `createFrameDropper`? This here is just a getter visibly only to tests, it'll return the current `frameDropper` (nullable) object.

I've updated the name of `initFrameDropper` to `createFrameDropper`: [VideoTrackTranscoder.java#L87](https://github.com/linkedin/LiTr/blob/a6d524823017dc2cf30ed85026e7d1ad3af4ace9/litr/src/main/java/com/linkedin/android/litr/transcoder/VideoTrackTranscoder.java#L87)